### ISSUE TITLE
Unexpected crash on last subvolume send

### ISSUE THREAD

**Issue by TH3L0N3WOLF** [digint/btrbk#22]
Upon transferring several snapshots to my backup volume, I received this error message.

```
At subvol /mnt/subvolroot/.snapshots/rtfs.20150531

ERROR: process died unexpectedly (btrbk v0.19.0)
Please contact the author: Axel Burri <axel@tty0.ch>

Stack Trace:
----------------------------------------
Use of uninitialized value in string ne at /usr/bin/btrbk line 2240, <FILE> line 26.
 at /usr/bin/btrbk line 119, <FILE> line 26.
	main::__ANON__("Use of uninitialized value in string ne at /usr/bin/btrbk lin"...) called at /usr/bin/btrbk line 2237
```
Believing there was something wrong with that particular rtfs snapshot, I deleted the snapshot for 20150531 and re-ran it. The same error came up with the previous snapshot as well, despite being transferred successfully.

**Comment by TH3L0N3WOLF** [comment]
Just to add to the mix, reverting to 0.18.0 fixes the issue.

**Comment by digint** [comment]
Thanks for reporting, the bug was in the statistics print function, which I *improved* (...) for 0.19.0. So don't worry, nothing is broken with your backups, btrbk was simply not able to print the statistics.

Fixed in db6a1a464add26ab36e22357c2f971ab044e868b
Fixed in btrbk-0.19.1
